cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -64,13 +64,13 @@ func main() {
 
 	swagger, err := util.LoadSwagger(flag.Arg(0))
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error loading swagger spec\n: %s", err)
+		_, _ = fmt.Fprintf(os.Stderr, "error loading swagger spec\n: %s", err)
```

Comment:
Can we not do _, _ = fmt.Fprintf. I realize lots of IDE's give warnings on uncosumed function output, but it's also pretty clear we're ignoring the result when we leave off the placeholder arguments